sdk/storage/AzureStorageBlob/Source/DataStructures/Transfer.swift

```diff
@@ -0,0 +1,96 @@
+// --------------------------------------------------------------------------
```

Comment:
Duplicated from `Source/Data/Transfer.swift`

Reply:
Probably a merge issue. Wasn't in the XCode project so it didn't cause warnings. I've removed the extra file. Thanks for the catch!